### DISCUSSION TITLE
How to export keypair?

### DISCUSSION THREAD

**Discussion by stokito** [openpgpjs/openpgpjs#1688] [Q&A] [answered]
Is it possible to export my keypair to .pgp file to save for a backup>
I just tried [Local-PGP](https://github.com/x0th/Local-PGP) browser extension but I can't import or export my pgp key. I wanted to send a PR but can't figure out if it's possible to do easily with the library.

**Comment by larabr** [comment, marked as answer]
If you have a `Key` object (from e.g. `openpgp.readKey`), you can simply call `key.armor()` and save that to a [text] file.
If it's a decrypted private key, and you want to encrypt it first, you can call `openpgp.encryptKey`.
If you're generating a key with `openpgp.generateKey`, you'll get an armored key pair by default (also encrypted, if you give a `passphrase), which you can again easily save to a file.

Does that work for you? Feel free to share more details if you have specific requirements for this backup.

**Reply by stokito** [reply]
thank you
